### ISSUE TITLE
CLI - sign and test Windows exec

### ISSUE THREAD

**Issue by sigalsax** [cyberark/conjur-api-python3#232]
During the bug bash, a bug was discovered that our company's security software doesn't let the exec run when we sign only the zip. The exec wasn't signed, only the zip. We need to test if we sign the exec if this will allow the exec to run.

DOD
- [x] Ask PO if the .zip itself needs to be signed for the CLI. If so, sign the exec and the zip
- [x] Sign _just_ the exec and then zip it and run on 3 or 4 Cyberark machines and see that both Antivirus and EPM doesn’t complain 
- [x] Run on a few Windows machines (non Cyberark machines) and see that it doesn’t trigger Antivirus
- [x] Answer the following questions: what does this mean for CLI executables that need to run on Linux machines? Can we just sign the zip? Can we zip the executables on non-windows machines without corrupting the exec? 
- [x] Build and sign a RHEL 8 executable and zip it and ensure it runs as well

**Comment by orenbm** [comment]
@InbalZilberman do we need to sign the zip that contains the exe file or only the exe itself?

**Comment by orenbm** [comment]
@sigalsax why do we need to run the exe on 3-4 machines and not 1? what are we trying to achieve?

**Comment by sigalsax** [comment]
@orenbm each machine is different so I want to make sure we don't get a false positive. If you feel confident in 2 then thats ok

**Comment by InbalZilberman** [comment]
Few clarifications: 
OSS products we do not need to be signed as a delivery (zip).
In the CLI the need to sign does not come from product but from OS requirements. For example in MAC we were not able to run the CLI without signing it.

**Comment by sigalsax** [comment]
Hi all, I checked the following flows on 3 different Cyberark machines and want to summarize my findings:
1. Executable is not signed
2. Executable is signed
3. Executable is signed and zipped (zip is not signed)

_Executable is not signed_
This flow was to run a sanity check and to have a base case. This check is used to ensure we we actually fixed the problem.
Result - Antivirus was alarmed!
[Screen Shot 2021-03-22 at 2.05.33 PM.png](https://images.zenhubusercontent.com/5cda5e517a32537dabbb1fda/a9602e4c-9862-48c5-b460-52b88354a47c)

_Executable is signed_
This flow is to make sure our signing works as expected as does not prompt anti-virus or EPM
Result: All is good!
![Screen Shot 2021-03-22 at 2.03.16 PM.png](https://images.zenhubusercontent.com/5cda5e517a32537dabbb1fda/558cb8bd-e360-424b-82e4-1dbacf58e1e8)

_Executable is signed and zipped (zip is not signed)_
This flow is to make sure the zipping does not corrupt the executable
Result: All is good!
![Screen Shot 2021-03-22 at 2.06.39 PM.png](https://images.zenhubusercontent.com/5cda5e517a32537dabbb1fda/a29e9713-fd75-4a76-9465-0e3b736c27b6)

I also checked zipping RHEL executables on Windows machines after the signing process and this did not corrupt the RHEL executable even though the zip happened on another OS
